gateway/config.go

```diff
@@ -19,7 +20,7 @@ type Config struct {
 	} `yaml:"frontend"`
 }
 
-type Server struct {
+type NetworkAddress struct {
```

Comment:
**NetworkAddress** means something else.
**Socket** is also a possibility... but that is less known. Just use Server to keep compatibility with Cortex [configuration](https://cortexmetrics.io/docs/configuration/configuration-file/#server_config)

What do you think?

Reply:
I honestly was not sure about that name. **Socket** did not resonate with me either. I think **ServerConfig** is best among all of the three.